XcodeServerSDK/Server Entities/Issue.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Issue.swift
+//  XcodeServerSDK
+//
+//  Created by Mateusz Zając on 04.08.2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+public class Issue: XcodeServerEntity {
+    
+    public enum Type: String {
+        case BuildServiceError = "buildServiceError"
+        // Looks like build service warnings are treated as warnings in case
+        // of type. In case of Issue Type they're Build Service Error
+        // case BuildServiceWarining = "warning"
+        case TriggerError = "triggerError"
+        case Error = "error"
+        case Warning = "warning"
+        case TestFailure = "testFailure"
+        case AnalyzerWarning = "analyzerWarning"
```

Comment:
If you want the whole list, this should be it:

```
var colors = {
    'error': 'rgb(200, 37, 6)',
    'warning': 'rgb(195, 151, 26)',
    'analyzerWarning': 'rgb(3, 101, 192)',
    'testFailure': 'rgb(200, 37, 6)',
    'resolved': 'rgb(160, 160, 160)',
    'buildServiceWarning': 'rgb(195, 151, 26)',
    'buildServiceError': 'rgb(200, 37, 6)',
    'triggerError': 'rgb(200, 37, 6)'
};
```
See `/Applications/Xcode-beta.app/Contents/Developer/usr/share/xcs/xcsd/templates/notifications.js` line `173`.